execution/gethexec/executionengine.go

```diff
@@ -442,17 +441,12 @@ func (s *ExecutionEngine) createBlockFromNextMessage(msg *arbostypes.MessageWith
 	}
 	statedb.StartPrefetcher("TransactionStreamer")
 	defer statedb.StopPrefetcher()
-	jst := msg.Message.Header.BlockJustification
-	var hotShotCommitment espresso.Commitment
-	if jst != nil {
-		hotShotCommitment = jst.Header.Commit()
-	}
 
 	block, receipts, err := arbos.ProduceBlock(
 		msg.Message,
 		msg.DelayedMessagesRead,
 		currentHeader,
-		&hotShotCommitment,
+		nil,
```

Comment:
Can we get rid of this parameter if it's always `nil`?

Reply:
We still need to provide the commitment here: https://github.com/EspressoSystems/nitro-espresso-integration/blob/integration/cmd/replay/main.go#L253